common/src/main/java/com/graphicsfuzz/common/util/CannedRandom.java

```diff
@@ -67,7 +67,7 @@ public boolean nextBoolean() {
 
   @Override
   public IRandom spawnChild() {
-    throw new UnsupportedOperationException("Child spawning not available");
```

Comment:
This is probably fine but did you mean to commit this?

Reply:
Yes, I did; apologies for not mentioning it in the commit message.